src/pages/project/{sanityProject.title}.jsx

```diff
@@ -1,12 +1,213 @@
+import CardPlaceholder from '../../images/card-placeholder.svg'
 
+
+const PrimaryContent = styled.div.attrs({
+  className: 'width-wrapper',
+})`
+  display: flex;
+  padding: 2rem 3rem 4rem 3rem;
+  justify-content: space-between;
+  align-items: flex-start;
+  align-self: stretch;
+  gap: 10%;
+
+  @media screen and (max-width: 426px) {
+    padding: 1rem 1rem 3rem 1rem;
+    flex-direction: column;
+    gap: 2rem;
+  }
+`
+const Left = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  gap: 1.5rem;
+  width: 39.5rem;
+  align-self: stretch;
+  @media screen and (max-width: 426px) {
+    padding: 0;
+    width: auto;
+  }
+`
+const Right = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 2rem;
+  @media screen and (max-width: 320px) {
+    flex-direction: column;
+    align-items: center;
+    align-self: stretch;
+  }
+`
+const CardDetailsColumn = styled.div`
+  display: flex;
+  align-items: flex-start;
+  gap: 1rem;
+  flex-direction: column;
+  @media screen and (max-width: 426px) {
+    align-items: center;
+    align-self: stretch;
+  }
+`
+const CardDetailsRow = styled.div`
+  display: flex;
+  align-items: flex-start;
+  gap: 1rem;
+    @media screen and (max-width: 426px) {
+  flex-direction: column;
+  align-items: center;
+  align-self: stretch;
+  }
+`
+const Bottom = styled.div`
+  background-color: var(--light-grey);
+  display: flex;
+  padding: 4rem 3rem;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  gap: 3rem;
+`
+const accordionList = [
+  {
+    position: 'UX Research',
+    role: [
+      'As a UX designer on this team you will support the project by iterating on interface designs based on research findings and then handing off designs to developers',
+    ],
+    responsibilities: [
+      'You will complete tasks in two week sprints and present designs to the team for feedback',
+    ],
+    preferreedSkills: [
+      'Familiar with Figma or other prototyping tools',
+      'Ability to collaborate with other designers, researchers, and developers',
+      'Must be comfortable giving and receiving constructive feedback',
+    ],
+  },
+
+  {
+    position: 'UX Design',
+    role: 'role description',
+    responsibilities: 'role responsibilities',
+  },
+]
 export default function Project({ data }) {
   console.log(data)
+  const project = data.sanityProject
+  let about = project.aboutThisPartner[0].children[0].text
+
+  let problem = project.problemToSolve[0].children[0].text
+
   return (
     <Layout>
-      <div>hi</div>
+      <PrimaryContent>
+        <Left>
+          <h2>About this Project</h2>
+          <p>{about}</p>
+          <h3>The Team</h3>
+          <p>Open Positions</p>
+          <Accordion accordionList={accordionList}></Accordion>
+          <h3>Tech Stack</h3>
+          <Button text={'Join us'} textBtn arrow></Button>
+        </Left>
+        <Right>
+          <h1>{project.title}</h1>
```

Comment:
Very nitpicky, but a couple things:
1. Let's make the project title `h2` because the `h1` is the Logo or more accurately the alt text of the logo: "Code for Chicago". Then of course adjust each heading tag accordingly.
2. Let's make the "Right" component before the "Left" one. I know visually that is not at all how it works, but think about if you were blind and learning about this project. You'd want to be told the title of the project first. Then the details. Not about the project, then the title of it. Then you'll have to use CSS to reverse the visual appearance of them to match the Figma. It's a bit of a pain, but it's our job to make designers markup semantic for non-browser user agents (braille readers, and screen readers and search engines etc).

Reply:
Thanks for pointing out these accessibility concerns. After addressing the header changes I noticed a few questions arise that affect the page's overall design and components. I'll Slack message you with specifics.